src/gurt/dlog.c

```diff
@@ -774,6 +774,7 @@ d_log_open(char *tag, int maxfac_hint, int default_mask, int stderr_mask,
 	uint64_t	log_size = LOG_SIZE_DEF;
 	int		pri;
 
+	memset(&mst, 0, sizeof(mst));
```

Comment:
Isn't it 0 initialized to begin with?

Reply:
Yes it will be because it's static.

Reply:
i am guessing whoever added this originally tried to cleanup things for repeated invocations of d_log_open(). in theory someone can open log, close it and reopen again

Reply:
That was my assumption as well.